tests/layer_tests/README.md

```diff
@@ -0,0 +1,30 @@
+# Layer tests
+
+This folder layer tests framework code and test files.
+
+## Getting Started
+
+#### Pre-requisites
+
+* OpenVINO should be configured as usual.
+
+#### Setup
+
+* Set up environment variables for layer tests:
+    ```bash
+    export MO_ROOT=PATH_TO_MO
+    ```
+    ```bash
+    export PYTHONPATH="path_to_openvino"/tests/layer_tests/:$PYTHONPATH
+    ```
+* If you need compare scoring results:
```

Comment:
inference isn't optional for now. Please, combine configuration with env keys above